src/components/Labelled/Labelled.tsx

```diff
@@ -26,6 +26,12 @@ export interface Props {
   children?: React.ReactNode;
   /** Visually hide the label */
   labelHidden?: boolean;
+  /** Character count in the labled input */
+  characterCount?: number;
+  /** Maximum character length for an input */
+  maxLength?: number;
```

Comment:
It seems a bit unclear what has a `maxLength`: the input or the label. This might want to be named `inputMaxLength`.